content/en/apps/examples/learning-care.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Learning and Care"
+linkTitle: "Learning & Care"
+weight: 
+description: >
+  An integration built to pilot the integrated workflows focused on CHW remote learning and care support for COVID-19. 
+keywords: oppia
+relatedContent: >
+  apps/features/integrations/oppia
+  apps/guides/integrations/oppia
+---
+
+The Learning & Care Apps in the Community Health Toolkit are designed to onboard community health workers remotely to new digital training tools, and help them learn new information and care delivery responsibilities through customized educational modules. 
+
+The modules can be deployed both (i) via the Android integration with the Academy App (powered by OppiaMobile) described in this documentation, and (ii) within the CHT core alone, to provide a seamless online & offline experience for CHWs, supervisors, and government partners.
+
+{{% alert title="Note" %}}
+It is important to consider the potential use cases for which a CHT core <> OppiaMobile integration is intended, as well as the digital landscape in which it is being designed and deployed. The CHT core <> OppiaMobile integration may be best suited for settings where both apps are already in use, given the inherent complexities in onboarding users remotely to two apps.
+{{% /alert %}}
+
+## Problem Being Addressed
+
+The Community Health Academy at Last Mile Health, with a range of content partners including Digital Medic, is building a library of open-access learning materials for community health workers.
+
+With the spread of COVID-19, it is imperative that access to learning, integrated with care protocols, routine task and care workflows, is available to CHWs without relying on face-to-face training.
+
+## Solution Overview
+
+The Community Health Academy and Medic Mobile set out to deliver a fully-integrated Learning and Care solution, with targeted COVID-19 learning experiences as a global public good.
+
+This integrated Learning & Care App leverages the **remote onboarding, task & scheduling, and target features** of the CHT core framework with the **curated, multimedia educational content** from the Academy’s COVID-19 library.
```

Comment:
can you link back to existing documentation site content for remote onboarding, tasks and targets?

Reply:
done!